gatsby-browser.js

```diff
@@ -12,8 +12,7 @@ function scrollToElementId(id) {
 
 export function onRouteUpdate({ location: { hash } }) {
   if (hash) {
-    if (!redirectLegacyRoutes()) {
-      window.setTimeout(() => scrollToElementId(hash), 10)
-    }
+    redirectLegacyRoutes()
+    window.setTimeout(() => scrollToElementId(hash), 10)
```

Comment:
This seems like a change in behavior. What’s the rationale?

Reply:
`redirectLegacyRoutes` does not return any value, so `if (!redirectLegacyRoutes())` would always evaluate to true.